src/tags/views.py

```diff
@@ -15,32 +16,55 @@
 
 from tags.forms import TagForm
 
-# from utilities.forms import TagForm, 
 from tenant.views import NonPublicOnlyViewMixin
 
-# from dal import autocomplete
+from django_select2.views import AutoResponseView
 from taggit.models import Tag
 from .models import get_quest_submission_by_tag, get_badge_assertion_by_tags, total_xp_by_tags
 
-# USE GENERIC utilities.views.ModelAutoComplete instead, might need this in the future if further customization is needed
-# class TagAutocomplete(autocomplete.Select2QuerySetView):
-#     """ https://django-autocomplete-light.readthedocs.io/en/master/taggit.html#view-example
-#     """
-#     def get_queryset(self):
-#         # Don't forget to filter out results depending on the visitor !
-#         if not self.request.user.is_authenticated:
-#             return Tag.objects.none()
-
-#         # order_by() added to prevent this warning:
-#         # UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'taggit.models.Tag'> QuerySet.
-#         qs = Tag.objects.all().order_by('name')
-
-#         if self.q:
-#             qs = qs.filter(name__istartswith=self.q)
+User = get_user_model()
 
-#         return qs
 
-User = get_user_model()
+class AutoResponseView(AutoResponseView):
+    """
+    View that handles requests from heavy model widgets.
+
+    The view only supports HTTP's GET method.
```

Comment:
This is not very informative, can you be more specific what this class is used for and why?

Reply:
fixed, see latest PR from me